README.md

```diff
@@ -39,15 +39,13 @@ helm install crossplane --namespace crossplane-system crossplane-alpha/crossplan
 ### Install the Crossplane CLI
 
 ```console
-curl -sL https://raw.githubusercontent.com/crossplane/crossplane-cli/master/bootstrap.sh | bash
+curl -sL https://raw.githubusercontent.com/crossplane/crossplane/release-0.13/install.sh | sh
 ```
 
 ## Install the Equinix Metal Provider
 
 ```console
-kubectl crossplane package install --cluster \
-  --namespace crossplane-system \
-  packethost/crossplane-provider-packet:v0.0.2 provider-packet
+kubectl crossplane install provider equinix/crossplane-provider-equinix-metal
```

Comment:
👍 
I followed https://github.com/packethost/crossplane-provider-equinix-metal/pull/30/files#diff-b335630551682c19a781afebcf4d07bf978fb1f8ac04c6bf87428ed5106870f5R34-R36 and ended up with 0.12.0. I needed to `helm repo update` first.

I also removed my existing /usr/local/bin/kubectl-crossplane* files before running the new crossplane cli install command.

Reply:
Likely okay to keep as is here right as these directions would theoretically be for a new user?

Reply:
Seems we'll need to update the Provider install instructions to ProviderConfig (with the new secret format).

The cluster/examples files will need revising.

I couldn't figure out how to install from kind's local images:
```
Events:
  Type     Reason         Age                 From                                 Message
  ----     ------         ----                ----                                 -------
  Warning  UnpackPackage  70s (x32 over 16m)  packages/provider.pkg.crossplane.io  cannot unpack package: failed to fetch package from remote: GET https://index.docker.io/v2/equinix/crossplane-provider-equinix-metal/manifests/v0.1.0: MANIFEST_UNKNOWN: manifest unknown; map[Tag:v0.1.0]
```

I've update packagePullPolicy to Never, but I think it wants a hostname somewhere 🤔

Reply:
This PR does not include updating to ProviderConfig and Provider will continue to work as is. This is expected behavior with the packagePullPolicy. It does not work like the old package manager, so you have to load images into the cache to have a pull policy of Never. Take a look at some of the integration tests for providers in Crossplane org if you would like more details.